functests/test_suite_test.go

```diff
@@ -24,10 +31,24 @@ func TestTest(t *testing.T) {
 	if junitPath != nil {
 		rr = append(rr, reporters.NewJUnitReporter(*junitPath))
 	}
-	RunSpecsWithDefaultAndCustomReporters(t, "Test Suite", rr)
+	RunSpecsWithDefaultAndCustomReporters(t, "Performance Addon Operators e2e tests", rr)
 }
 
 var _ = BeforeSuite(func() {
-	//clients.Setup()
-	// Add here the setup for additional features
+	cs := testutils.NewClientSet("")
```

Comment:
could be clients.New() (with a separate module). In that way only const remains in testutils which can then be switched to testconsts

Reply:
will do

Reply:
done